draft-ietf-add-ddr.md

```diff
@@ -343,14 +348,16 @@ resolvers with a large number of referring IP addresses.
 ## Server Name Handling
 
 Clients MUST NOT use "resolver.arpa" as the server name either in the TLS
-Server Name Indication in TLS ({{?RFC8446}}) for DoT or DoH; or in the URI host
-for DoH requests.
+Server Name Indication in TLS ({{?RFC8446}}) for DoT or DoH connections,
+or in the URI host for DoH requests.
 
-Designated DoH resolvers that support authenticated discovery MUST accept
-both of the following as the URI host in requests:
+Designated DoH resolvers that support authenticated discovery for clients
+that only know an IP address MUST accept both of the following as the
+URI host in requests:
 
 - the IP address of designating Unencrypted Resolver
-- the TargetName in the ServiceMode SVCB record
+- the TargetName in the ServiceMode SVCB record, which is the
+hostname for which the client looked up address (A and AAAA) records
```

Comment:
Minor sugegstion: + " if the SVCB response did not contain additional A/AAA records or IP address hints."

(either way, needs the end punctuation though)

Reply:
Fair! I've changed the text to say that it's the hostname that maps to the address records. Technically, the A/AAAA records need to be grabbed even if the SVCB record has hints, but I removed the text that says that the client queried for these records (since they may have been present in additional records).